packages/aws-rfdk/lib/deadline/scripts/python/client-rq-connection.py

```diff
@@ -168,7 +168,15 @@ def configure_deadline( config ):
 
             call_deadline_command(['SetIniFileSetting', 'ProxySSLCA', cert_path])
             call_deadline_command(['SetIniFileSetting', 'ClientSSLAuthentication', 'NotRequired'])
-            repo_args.append(cert_path)
+            
+            # Validate Deadline connection
+            print("Testing Deadline connection...")
+            stdout.flush()
+            try:
+                call_deadline_command(['GetJobs'])
```

Comment:
We may want to change this to a command that scales better. In the case of an upgrade, there may be many jobs